src/firmware/main/math/polynomial_2d_test.cpp

```diff
@@ -4,6 +4,7 @@ extern "C"
 }
 
 #include <gtest/gtest.h>
+#include <math.h>
```

Comment:
Duplicate `include` with line 9? I'd say remove line 9 and keep your change since `<>` is more correct here.

Reply:
fixed